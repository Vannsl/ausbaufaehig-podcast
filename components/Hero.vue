<template>
  <section class="hero bg-black" />
</template>

<script>
export default {
  name: "Hero"
};
</script>

<style lang="scss" scoped>
.hero {
  background: url("https://res.cloudinary.com/vannsl-io/image/upload/c_scale,w_2400,q_40/v1548597279/IMG_7750.jpg");
  background-position: center;
  background-size: cover;
  min-height: calc(100vh - 0px);
}
</style>
