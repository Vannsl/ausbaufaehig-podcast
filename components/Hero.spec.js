import { shallowMount } from "@vue/test-utils";
import Hero from "./Hero";

describe("Hero", () => {
  const wrapper = shallowMount(Hero);

  it("renders the correct markup", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
