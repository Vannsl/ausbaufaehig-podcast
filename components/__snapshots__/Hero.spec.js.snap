// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders the correct markup 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
